src/app/attendance/actions/getAttendance.ts

```diff
@@ -7,7 +7,7 @@ import { getMembers } from "@/app/members/actions/getMembers";
 export const getAttendance = async () => {
   const supabase = await getSupabaseClient();
 
-  const membersResponse = await getMembers();
+  const membersResponse = await getMembers("");
```

Comment:
@nadirhussain786 why you passing string?

Reply:
@muhammad-bilal-ali-khokhar  Actually, I added the search functionality, which is why I passed an empty string as the payload.